docs-source/content/userguide/cicd/how-to-copy-domains.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "How to copy domains"
```

Comment:
Would this be more appropriate to call "How to create primordial domain layer" along with a description to match?

Reply:
I am trying to keep the titles as brief as possible so the table of contents is not full of long ugly things